core/domain/question_domain_test.py

```diff
@@ -60,25 +61,88 @@ def test_change_dict_with_wrong_cmd(self):
         """Test to verify __init__ method of the Question Change object
         when change_dict is with wrong cmd value.
         """
-        self.assertRaises(
-            Exception,
+        self.assertRaisesRegexp(
+            utils.ValidationError,
+            'Command wrong is not allowed',
             callableObj=question_domain.QuestionChange,
             change_dict={'cmd': 'wrong', }
         )
 
+    def test_change_dict_with_missing_attributes_in_cmd(self):
+        """Test to verify __init__ method of the Question Change object
+        when change_dict is with missing attributes in cmd.
+        """
+        self.assertRaisesRegexp(
+            utils.ValidationError,
+            'The following required attributes are present: new_value',
+            callableObj=question_domain.QuestionChange,
```

Comment:
Neat, didn't know about this API!

Reply:
Me too :P This was already present in the codebase :)